source/vibe/web/rest.d

```diff
@@ -368,17 +349,15 @@ class RestInterfaceClient(I) : I
 	}
 
 	/// ditto
-	final @property void requestFilter(RequestFilter v) {
+	final @property void requestFilter(RequestFilter v)
+	{
 		m_requestFilter = v;
-		mixin (generateRestInterfaceSubInterfaceRequestFilter!I());
+		foreach (i, SI; Info.SubInterfaceTypes)
+			m_subInterfaces[i].requestFilter = v;
 	}
 
-	//pragma(msg, "subinterfaces:");
-	//pragma(msg, generateRestInterfaceSubInterfaces!(I)());
-	mixin (generateRestInterfaceSubInterfaces!I());
-
 	//pragma(msg, "restinterface:");
-	mixin RestClientMethods!I;
+	mixin(generateRestClientMethods!I());
```

Comment:
What I'm really looking to kill is actually `generateModuleImports`. And yeah, taking a second look at the code, it looks like it's not a problem.